studio.assetmanager.ams.yaml

```diff
@@ -0,0 +1,77 @@
+app-id: studio.assetmanager.ams
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: ams
+finish-args:
+  # X11 performance
+  - --share=ipc
+  # Access to network
+  - --share=network
+  # Access to X11
+  - --socket=x11
+  # Access to wayland
+  - --socket=wayland
+  # Audio Access
+  - --socket=pulseaudio
+  # Access to all files
+  - --filesystem=host
```

Comment:
this is very broad

Reply:
Yeah, unfortunately, people can put their UE projects, asset vaults, & unreal engines ANYWHERE on the machine.   

On my machine I have two vaults in my main home/document folder (mine, & Epic default vault) and another one on a my unreal drive mount point.   My UE's are on a that same unreal mount point, and my projects are also on the same unreal mount point.   I don't see any easy way around this as there is no "set" locations for anything when it comes to Unreal.